test/test_history_saving.vader

```diff
@@ -67,16 +68,27 @@ Given foobar (Some imaginary filetype):
 Execute(History should be set when commands are run):
   AssertEqual 'foobar', &filetype
 
-  call ale#Lint()
-  call ale#engine#WaitForJobs(2000)
```

Comment:
It looks like you're undoing some changes to some files here. It looks like you used `git merge`. I typically recommend using only `git rebase` for pull requests. Seeing as you've already used `git merge`, just use `git merge` again with `master`, remove the changes to other files that you don't want, and I'll just squash everything into one commit at the end.

Reply:
That's one way to do it. As long as the patch doesn't change some unrelated files, it's good.